src/modules/client/components/clientAlerts/ClientAlert.tsx

```diff
@@ -0,0 +1,52 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import {
+  parseAndFormatDate,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import { ClientAlertFieldsFragment } from '@/types/gqlTypes';
+
+export type ClientAlertType = {
+  alert: ClientAlertFieldsFragment;
+  clientName: string;
+  showClientName?: boolean;
+};
+
+interface ClientAlertProps {
+  clientAlert: ClientAlertType;
+}
+const ClientAlert: React.FC<ClientAlertProps> = ({ clientAlert }) => {
+  const alert = clientAlert.alert;
+  const clientName = clientAlert.clientName;
+  const showClientName = clientAlert.showClientName;
+  const priority = alert.priority || 'low';
```

Comment:
```suggestion
  const { alert, clientName, showClientName, priority } = clientAlert;
```

There is no need to set a default for the `priority` since it's non-nullable from graphql